pkg/service/record/record.go

```diff
@@ -68,8 +70,8 @@ func (r *recorder) CaptureTraffic(path string, proxyPort uint32, appCmd, appCont
 	case <-stopper:
 		return
 	default:
-		// load the ebpf hooks into the kernel
-		if err := loadedHooks.LoadHooks(appCmd, appContainer, 0, ctx, filters); err != nil {
+		// Load the ebpf hooks into the kernel
+		if err := loadedHooks.LoadHooks(params.AppCmd, params.AppContainer, 0, ctx, params.Filters); err != nil {
```

Comment:
updating the variable names seems unnecessary as no linter or anything is affected

Reply:
Only the cases of the variables have been changed to ensure consistency with the struct. Do you want them camelCased?